modules/basics/src/main/resources/com/opengamma/strata/config/base/IborIndexData.csv

```diff
@@ -109,3 +109,7 @@ NOK-NIBOR-1M,NOK,true,Act/360,NOOS,2,NOOS,NOOS,1M,LastBusinessDay,12:00,Europe/O
 NOK-NIBOR-2M,NOK,true,Act/360,NOOS,2,NOOS,NOOS,2M,LastBusinessDay,12:00,Europe/Oslo
 NOK-NIBOR-3M,NOK,true,Act/360,NOOS,2,NOOS,NOOS,3M,LastBusinessDay,12:00,Europe/Oslo
 NOK-NIBOR-6M,NOK,true,Act/360,NOOS,2,NOOS,NOOS,6M,LastBusinessDay,12:00,Europe/Oslo
+,,,,,,
+AUD-BBSW-3M,AUD,true,Act/365F,AUSY,1,AUSY,AUSY,3M,LastBusinessDay,11:00,Australia/Sydney
```

Comment:
BBSW exists for 1m to 6m (see http://www.afma.com.au/data/bbsw). We should have all of them in the list.